addons/centered/src/components/Centered.svelte

```diff
@@ -0,0 +1,34 @@
+<div class="style">
+  <div class="inner-style">
+    <svelte:component this="{Story}" {...data} />
+  </div>
+</div>
+
+<style>
+  .style {
```

Comment:
I can, but it will make the svelte template "worse". The style attribute doesn't work the same way as in react or vue, so I can't just pass the object in there.

I'll have a go, and see what you think.

Reply:
I'm adding a small helper function to convert it to plain CSS strings before applying it inline in the component. Not very pretty but solves the problem. It doesn't make the component 'worse', so I guess there's that.